client/packages/system/src/Encounter/DetailView/DetailView.tsx

```diff
@@ -244,16 +150,19 @@ export const DetailView: FC = () => {
   } = useSaveWithStatusChange(
     saveData,
     data as unknown as EncounterSchema,
-    updateEncounter,
-    nextEncounterModal.toggleOn
```

Comment:
Changed this openModal call from being another param to this hook, to being a callback of `saveWithStatusChange`, see below